src/features/antora/fastFindFiles.ts

```diff
@@ -0,0 +1,50 @@
+import vscode from 'vscode'
+
+let findFiles2Available : boolean | undefined
+
+const exclude : string[] = [
```

Comment:
excludes and includes adjusted for performance even if we use the 'standard' findFiles

Reply:
I think it's better to use the `files.exclude` configuration as mentioned in the API documentation: https://code.visualstudio.com/api/references/vscode-api#workspace

``**/.DS_Store` and `**/node_modules` are probably fine but I won't be surprised if someone is putting documentation in a directory named `vendor` or `build` (or even `coverage`)